### PR TITLE
Garnett - meta position

### PR DESCRIPTION
**Pull request by sndrs**
## What does this change?

- reverts changes to article header for article, re-aligning meta with media
- toggles tone/pillar classes depending on whether we're in garnett or non-garnett mode

### CODE REVIEW

File: article/app/views/fragments/articleBody.scala.html [hunk 1]
```diff
@@ -23,12 +24,14 @@
         class="@RenderClasses(Map(
             "has-feature-showcase-element" -> (article.tags.isFeature && article.elements.hasShowcaseMainElement),
             "paid-content" -> isPaidContent,
-            "paid-content--white" -> (isPaidContent && ActiveExperiments.isParticipating(CommercialPaidContentTemplate))
+            "paid-content--white" -> (isPaidContent && ActiveExperiments.isParticipating(CommercialPaidContentTemplate)),
+            (s"content--pillar-${article.metadata.pillar.nameOrDefault}", experiments.ActiveExperiments.isParticipating(experiments.Garnett)),
+            (s"content--type-${article.metadata.designType.nameOrDefault}", experiments.ActiveExperiments.isParticipating(experiments.Garnett)),
+            ("tonal", !experiments.ActiveExperiments.isParticipating(experiments.Garnett)),
+            (s"tonal--${toneClass(article)}", !experiments.ActiveExperiments.isParticipating(experiments.Garnett)),
```
Comment (TBonnin) [inline article/app/views/fragments/articleBody.scala.html]: why stop using the arrow notation `key -> value` and introduce the tuple notation `(key, value)` in this `Map`

Comment (NataliaLKB) [reply article/app/views/fragments/articleBody.scala.html]: We should definitely stick to a consistent syntax. I would probably stick to using `key -> value` syntax here.

**Comment by NataliaLKB** [comment]
@sndrs this mostly looks good! Just make sure to use a consistent syntax for map

File: article/app/views/fragments/articleBody.scala.html (hunk 1 above)
Comment (sndrs) [reply article/app/views/fragments/articleBody.scala.html]: no good reason - bad copy/paste. will fix

**Comment by sndrs** [comment]
i think should build now, fixed those map inconsistencies – @NataliaLKB would you be ok to merge it if it's all 🥗?

**Pull request merged**